src/azure-cli/setup.py

```diff
@@ -132,7 +132,7 @@
     'knack~=0.6,>=0.6.3',
     'mock~=2.0',
     'paramiko>=2.0.8,<3.0.0',
-    'prompt_toolkit~=1.0,>=1.0.15',
+    'prompt_toolkit>=1.0.15',
```

Comment:
Unless we're limiting ourselves to surface area available in both major versions 1 & 2, we should use `prompt_toolkit~=2.0`.

Reply:
I'll change it now.  FWIW, my reasoning in keeping just the `>=` constraint is that it looks like the code doesn't use anything specific for v2, but I might be wrong.